packages/frontend/src/routes/router.tsx

```diff
@@ -20,12 +21,15 @@ import { CreateUserPage } from 'src/pages/CreateUserPage/CreateUserPage';
 import { EditUserPage } from 'src/pages/EditUserPage/EditUserPage';
 import { EventDetailsPage } from 'src/pages/EventDetailsPage/EventDetailsPage';
 
+import { CreateVolunteerPage } from 'src/pages/CreateVolunteerPage/CreateVolunteerPage';
+
 const routes = (
   <Route element={<App />}>
```

Comment:
@epmtesla I need your opinion) Should this page be here? I'm concerned, because:
1) Will require auth to open, which won't be the case for telegram bot app
2) Shows menu at the top, which should not be visible in the telegram bot

Should we deal with it in this task or we have some other to do it?